src/common/addWithEnter.js

```diff
@@ -0,0 +1,23 @@
+import { List } from './lists.js';
+
+const addWithEnter = () => {
+  const description = document.getElementById('input-todo').value;
+  const list = new List();
+  list.add(description, false);
+  document.getElementById('input-todo').value = '';
+  const storedTodoList = JSON.parse(localStorage.getItem('toDolist'));
+  if (localStorage.getItem('toDolist') == null) {
+    localStorage.setItem('toDolist', JSON.stringify(list.list));
+    /* eslint-disable */
+    location.reload();
+    /* eslint-enable */
+  } else {
+    const newList = storedTodoList.concat(list.list);
+    localStorage.setItem('toDolist', JSON.stringify(newList));
+    /* eslint-disable */
+    location.reload();
+    }
+   }
+
+/* eslint-disable */
+export { addWithEnter }
```

Comment:
- [ ] Kindly make sure you fix all your `linters` errors instead of disabling them as you did for best practices and clean code.